js-peer/src/lib/libp2p/index.ts

```diff
@@ -67,72 +80,90 @@ export async function startLibp2p() {
     // The app-specific go and rust peers use WebTransport and WebRTC-direct which have ephemeral multiadrrs that change.
     // Thus, we dial them using only their peer id below, with delegated routing to discovery their multiaddrs
     // peerDiscovery: [
-      // bootstrap({
-        // list: [
-          // '12D3KooWFhXabKDwALpzqMbto94sB7rvmZ6M28hs9Y9xSopDKwQr'
-          // WEBRTC_BOOTSTRAP_NODE,
-          // WEBTRANSPORT_BOOTSTRAP_NODE,
-          // '/dnsaddr/bootstrap.libp2p.io/p2p/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN',
-          // '/dnsaddr/bootstrap.libp2p.io/p2p/QmQCU2EcMqAqQPR2i9bChDtGNJchTbq5TbXJJ16u19uLTa',
-          // '/dnsaddr/bootstrap.libp2p.io/p2p/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb',
-          // '/dnsaddr/bootstrap.libp2p.io/p2p/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt',
-        // ],
-      // }),
+    // bootstrap({
+    // list: [
+    // '12D3KooWFhXabKDwALpzqMbto94sB7rvmZ6M28hs9Y9xSopDKwQr'
+    // WEBRTC_BOOTSTRAP_NODE,
+    // WEBTRANSPORT_BOOTSTRAP_NODE,
+    // '/dnsaddr/bootstrap.libp2p.io/p2p/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN',
+    // '/dnsaddr/bootstrap.libp2p.io/p2p/QmQCU2EcMqAqQPR2i9bChDtGNJchTbq5TbXJJ16u19uLTa',
+    // '/dnsaddr/bootstrap.libp2p.io/p2p/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb',
+    // '/dnsaddr/bootstrap.libp2p.io/p2p/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt',
+    // ],
+    // }),
     // ],
     services: {
       pubsub: gossipsub({
+        emitSelf: false,
         allowPublishToZeroTopicPeers: true,
         msgIdFn: msgIdFnStrictNoSign,
         ignoreDuplicatePublishError: true,
+        tagMeshPeers: true,
       }),
+      dht: kadDHT({
```

Comment:
What's the reason for enabling the DHT in the browser again? 

I added delegated routing to avoid running a DHT client in the browser, but if there's a good reason, we can enable. 

Also see https://github.com/libp2p/universal-connectivity/pull/113 for more background on why this app relies on the IPFS DHT.